docs/Concepts/Merge.md

```diff
@@ -62,3 +62,12 @@ After The Merge, in addition to validators earning rewards for performing [valid
 layer transactions.
 
 You can [prepare Teku for The Merge](../HowTo/Prepare-for-The-Merge.md).
+
+## After The Merge
+
+After the merge of a network is complete, a single 'node' becomes a combination of both the execution client and the consensus client. 
+If either component is not available, the node will not be available on the network.
```

Comment:
I don't think that "available on the network" is the right phrase here. The CL will be connected to the network and have peers, but if the EL is not available at all, then it can't import blocks (not even optimistically) so it will be unable to follow the chain head correctly. Once the EL is available again it will then import those blocks and catch back up to the chain head.

Reply:
Fair call, I was mostly trying to get a starting point covering the intent of what to say and hoping someone could make it pretty...

Reply:
Maybe something like "node will not be a full-fledged member of the network"